app/models/profile.rb

```diff
@@ -13,4 +13,8 @@ class Profile < ApplicationRecord
   def to_s
     name
   end
+
+  def attending?(event)
+    event_attendees.where(event: event).any?
```

Comment:
Is this better than `event.event_attendees.present?`

Reply:
Yes, because it checks if the profile is attending the event, not just if the event has attendees.